src/database.ts

```diff
@@ -976,6 +1017,59 @@ class Database extends GrpcServiceObject {
       callback!
     );
   }
+
+  /**
+   * Retrieves the restore information of the database.
+   *
+   * @see {@link #getMetadata}
+   *
+   * @method Database#getRestoreInfo
+   * @returns {Promise<IRestoreInfoTranslatedEnum | undefined>}
+   *     when resolved, contains the restore information for the database if it exists,
```

Comment:
I don't believe the part about returning undefined if the database doesn't exist is accurate.

Reply:
Agree, it will only return something meaningful if it was restored from a backup.